src/styles/main.scss

```diff
@@ -0,0 +1,76 @@
+$middle: 50%;
+$handsSize: 10px;
+
+.page {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+@mixin clock-hand {
+  position: absolute;
+
+  bottom: $middle;
+  left: $middle;
```

Comment:
you need add calculations position with 50% from arrow, for correct display

Reply:
good point, now the arrows are not in the middle in`stopwatch__middle`

<img width="76" alt="Screenshot 2022-04-13 at 10 05 45" src="https://user-images.githubusercontent.com/37452763/163119393-385dbb2c-e19b-4669-960f-d030d706dbcc.png">